docs/tutorials/exporting_results.ipynb

```diff
@@ -4,7 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>It is good. Can you mention just a bit that you are saving files as parquet files in a directory structure.</p><p> </p><p>After that, please merge </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/318/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Of course, I just added a brief overview of the directory structure!